varnish_request_exporter.go

```diff
@@ -173,8 +172,8 @@ func parseMappings(input string) (mappings []str_mapping, err error) {
 	return
 }
 
-func buildVarnishncsaArgs(httpHost string) (args []string) {
-	args = make([]string, 2, 4)
```

Comment:
Shouldn't the capacity be 6 here, since you'd be adding `-n` and the instance name?